specification/edgeorder/resource-manager/Microsoft.EdgeOrder/preview/2020-12-01-preview/edgeorder.json

```diff
@@ -343,19 +343,19 @@
           "nextLinkName": "nextLink"
         },
         "x-ms-examples": {
-          "ListOrderCollectionsAtSubscriptionLevel": {
-            "$ref": "./examples/OrderCollectionListAtSubscriptionLevel.json"
+          "ListOrderAtSubscriptionLevel": {
+            "$ref": "./examples/ListOrderAtSubscriptionLevel.json"
           }
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.EdgeOrder/orders": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EdgeOrder/orderItems": {
```

Comment:
Can you summarize what the actual changes are?  What resource types are being renamed and what is functionally different in this new version?

Reply:
Major changes involves

1. Renaming resources, As per ARM recommendation we renamed our tracked resource orders to orderItems and proxy resource ordercollections to orders [To briefly summarize the context, we created orders which groups orderItems only for customer viewing experience]
2. We made Ordercollections [renamed to orders] as proxy resource as it doesnt have any role in our service other than providing grouping experience. Also, we take care of creating ordercollections[orders] internally and associate with order item.